tests/data_context/migrator/conftest.py

```diff
@@ -1,283 +0,0 @@
-from typing import Dict, Final, List, Optional, Tuple, Union
```

Comment:
Rather than deleting the migrator (which caused import cycles), I'm just deleting tests so we can move forward without issue. Same comment applies to everything in this directory